step2-02/README.md

```diff
@@ -47,11 +47,11 @@ const MyComponent = () => {
 };
 ```
 
-## Customizing Behavior of Individual Component
+### Customizing Behavior of Individual Components
```

Comment:
It's a little sad to see this using legacy customization methods instead of slots--guessing that's because slots support is still very much a work in progress? (If you end up needing more content, you could add sort of a preview of slots.)

Reply:
I'm going to plug Slots but best person to do the talk would be @JasonGore, not me 👍

Reply:
Hmm, I do have links to [Slot content](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Slots-&-Tokens) here that could be referred to. It definitely needs to be updated and consolidated, though.

Reply:
Either way, I think the idea of "render functions" in general is worth introducing. Whether it's using legacy props or slots, the idea of a render function is the biggest hump I think.